docs/installation.rst

```diff
@@ -376,6 +378,8 @@ Using git
 This uses latest, unreleased development code from git.
 While we try not to break master, there are no guarantees on anything.
 
+Install the dependencies for your OS, for example :ref:`debian-ubuntu`
+
```

Comment:
this documentation was assuming that people just read it top-down, as usual, but some people directly jump into the middle of it and assume all will be fine...

pip (line 323) as well as git (line 373) are source-based installations of borg and thus have the requirements as documented in the "From source" section (line 143+).

So, if you want to point people there, the correct link target is in line 143 and the pointer needs to be added to the pip section also.

Reply:
> this documentation was assuming that people just read it top-down, as usual, but some people directly jump into the middle of it and assume all will be fine...

The problem of this (and many other) documentation is that it assumes too much.

Naturally, if I want to install a dev-system, I jump right to the dev-system section (and I assume that all be fine, e.g. that authors of the docs managed to have all relevant info there).

> So, if you want to point people there, the correct link target is in line 143 and the pointer needs to be added to the pip section also.

I write documentation (especially for developers) "straight-to-the-point", similar here. Pointing to "143" results in (me, and others, i bet) skipping again fast-forward until they see their OS of interest.

---
my most important note was:

>>(dev environment setup can be further streamlined, if this is of interest)

Are you interested in streamlining the documentation/installation process? Or just in "micromanaging" some doc changes?

Reply:
> e.g. that authors of the docs managed to have all relevant info there

We won't duplicate the install docs into the developers docs. If we don't have that already, add a pointer from there to here.

> Pointing to "143" results in (me, and others, i bet) skipping again fast-forward until they see their OS of interest.

If you skip from 143 to the OS you use, you should be fine. Just pointing to the debian/ubuntu section is only helpful IF you use these.

Reply:
> If you skip from 143 to the OS you use, you should be fine.

You should really not write / manage documentation.

Reply:
see below for more.